gorush/notification_apns.go

```diff
@@ -98,7 +98,12 @@ func InitAPNSClient() error {
 			}
 		}
 
-		if ext == ".p8" && PushConf.Ios.KeyID != "" && PushConf.Ios.TeamID != "" {
+		if ext == ".p8" {
+			if PushConf.Ios.KeyID == "" || PushConf.Ios.TeamID == "" {
+				msg := "You should provide ios.KeyID and ios.TeamID for P8 token"
+				LogError.Error(msg)
+				return errors.New(msg)
+			}
```

Comment:
Only check for `.p8` extension?

Reply:
sorry, yes, it needed only for p8

Reply:
@slimus Thanks. https://stackoverflow.com/questions/40621097/what-is-the-keyid-when-using-apns-push-service